src/authentication/authentication.service.ts

```diff
@@ -21,10 +22,18 @@ export class AuthenticationService {
     return this.usersService.create({
       name: signUpData.name,
       email: signUpData.email,
+      phoneNumber: signUpData.phoneNumber,
       password: hashedPassword,
     });
   }
 
+  async updatePhoneNumber(
```

Comment:
`updatePhoneNumber` has nothing to do with authentication. It should be in the `UsersService` instead